index.bs

```diff
@@ -617,8 +621,9 @@ To <dfn>release the lock</dfn> |lock|:
 
 1. [=Assert=]: these steps are running on the [=lock task queue=].
 1. Let |origin| be |lock|'s [=lock-concept/origin=].
+1. Let |name| be |lock|'s associated [=resource name=].
 1. [=list/Remove=] [=lock-concept|lock=] from the |origin|'s [=held lock set=].
-1. [=Process the lock request queue=] for |origin|.
+1. [=Process the lock request queue=] associated with |name| in the |origin|'s [=lock request queue set=].
```

Comment:
Ideally this would get |queue|, then invoke the algorithm with |queue| and |origin|